services/httppost/service.go

```diff
@@ -27,25 +27,55 @@ func NewService(c Configs, l *log.Logger) *Service {
 	return s
 }
 
+// Only one of name and url should be non-empty
+type Endpoint struct {
+	s      *Service
+	name   string
+	url    string
```

Comment:
> This way all endpoints are the same and just reference a URL and a set of headers.

I like this idea, but I think we might need the name for the case when an endpoint hasn't been defined, but is used in a tickscript. When the task in initialized and the endpoint doesn't currently exist and then we create the endpoint. When the configuration is updated how do we know which `httppost.Endpoints` need to have their values updated? I could be over thinking this.

> Also, where are the headers? Are they in the config struct still?

Yeah they're only in the config struct.